package.json

```diff
@@ -136,13 +136,14 @@
     "karma-webpack": "^1.8.0",
```

Comment:
Review is not for this line of code but for lines 87-93. I had to change to the following this get it to work:
```
  "scripts": {
    "postinstall": "\"\"./node_modules/.bin/bower\"\" install",
    "pretest": "npm run lint",
    "test": "\"\"./node_modules/.bin/karma\"\" start --single-run --no-auto-watch",
    "autotest": "\"\"./node_modules/.bin/karma\"\" start --no-single-run --auto-watch",
    "lint": "\"\"./node_modules/.bin/eslint\"\" --ext .js,.jsx --plugin react src"
  },
```